app/src/main/java/Activities/ViewProblemActivity.java

```diff
@@ -25,29 +27,88 @@
 
-    private TextView view_problem_title_text;
     private int position;
     private ArrayList<Problem> problems;
+    private Problem currentProblem;
+    private String userId;
+    private BrowseUserProblemsController problemController = new BrowseUserProblemsController();
+
+    @Override
+    protected void onStart() {
+        super.onStart();
+
+    }
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_view_problem);
 
-        this.view_problem_title_text = (TextView)findViewById(R.id.view_problem_title_id);
-
+        //Extract selected problem object through intent and index of problem list
         Intent intent = getIntent();
         this.position = intent.getIntExtra("position",0);
+        this.userId = intent.getStringExtra("USERIDEXTRA");
+        this.problems = problemController.getProblemList(ViewProblemActivity.this,this.userId);
+        this.currentProblem = this.problems.get(this.position);
+
+        //initialize TextViews and Buttons
+        this.view_problem_title_text = (TextView)findViewById(R.id.view_problem_title_id);
+        this.view_problem_date_text = (TextView)findViewById(R.id.view_problem_date);
+        this.view_problem_description_text = (TextView)findViewById(R.id.view_problem_description);
+        this.view_problem_numRecords_text = (TextView)findViewById(R.id.view_problem_numRecords);
+        this.setReminderButton = (Button)findViewById(R.id.view_problem_setReminder);
+        this.viewRecordsButton = (Button)findViewById(R.id.view_problem_viewRecords);
+        this.viewMapButton = (Button) findViewById(R.id.view_problem_viewMap);
+        this.viewSlideshowButton = (Button)findViewById(R.id.view_problem_viewSlideshow);
+
+        //set text for TextViews
+        String tempString = "Problem: " + this.currentProblem.getTitle();
+        this.view_problem_title_text.setText(tempString);
+
+        tempString = "Date: " + this.currentProblem.getDate().toString();
+        this.view_problem_date_text.setText(tempString);
+
+        tempString = "Description: " + this.currentProblem.getDescription();
+        this.view_problem_description_text.setText(tempString);
+
+        tempString = "Number of Records: " + Integer.toString(this.currentProblem.getRecordCount());
+        this.view_problem_numRecords_text.setText(tempString);
     }
 
     @Override
     protected void onResume() {
         super.onResume();
 
-
+    }
+    public void onSetReminderClick(View v){
+        //TODO create intent and create activity class for setting alarm
+    }
+    public void onViewMapClick(View v){
+        //TODO create intent and create activity class for viewing map
+    }
+    public void onViewRecordsClick(View v){
+        Intent intent = new Intent(this, ViewRecordsActivity.class);
+        intent.putExtra("USERIDEXTRA",this.userId);
+        intent.putExtra("position",this.position);
```

Comment:
You should also putExtra problem title/id, so that the record knows which user's problem is being passed